code/_onclick/hud/screen_objects/base_screen_objects.dm

```diff
@@ -1090,6 +1090,12 @@ obj/screen/fire/DEADelize()
 		if (G.active && G.overlay)//check here need if someone want call this func directly
 			overlays |= G.overlay
 
+	if(istype(H.back,/obj/item/weapon/rig))
```

Comment:
A request;
Mobs, or at least humans, have a wearing_rig var.
It currently doesn't work, its not getting populated when you wear one

Find out why, fix it, and then check it instead of this check here

Reason being, the design spec for rigs doesn't require them to be on the back. There's a future possibility of rigs that come in a belt, or a shirt, or a tie or whatever. Lets try to avoid checking a specific slot